WISE-Community/WISE
# Fix discussion component slowdown after 20 responses

To test:
1. Log in as a student and go to a step with a discussion component in a run.
1. Add responses. Simple responses like "1", "2", "3"... will suffice.
1. Keep going. Add some comments (replies).
1. Verify that posting and commenting don't cause much of a delay and that scrolling doesn't slow down.
1. Log in as another student in the same run and add responses to the discussion component.
1. Log in as a teacher and open the teacher tools for the run.
1. View the discussion component in the step grading view.
1. Make sure discussion posts for each workgroup show up correctly.

Resolves #2184.
